### PR TITLE
Feature/new auction statuses

### PR DESCRIPTION
**Pull request by Velenir**
New auction statuses and conditions for checking auction state

### CODE REVIEW

**Comment by W3stside** [comment]
@Velenir heading to an token pair auction index that is `PLANNED` stage has confusing aspects.

Shows `deposit confirmed` in status bar when that is sometimes not true - I would remove this via a conditional check or something else -  maybe we can add more concise wording to describe what is happening

_To Test_: `npm run addTokenPairs` -> head to `.../auction/ETH-GNO-2`

**Comment by Velenir** [comment]
Yes, progress bar still basically counts steps between statuses [STATUS.PLANNED--ACTIVE--ENDED]. As it was at the very beginning.
That needs to be changed, but I'd like to dedicate a separate PR to that later.

**Comment by W3stside** [comment]
@Velenir if you make the ticket and assign yaself I'll approve it :D

**Review by W3stside — APPROVED** [review]
